tokens/movistar.json

```diff
@@ -1239,34 +1239,34 @@
       "description": "darkModeGrey2"
     },
     "textAppBar": {
-      "value": "{palette.darkModeGrey5}",
+      "value": "{palette.darkModeGrey4}",
       "type": "color",
-      "description": "darkModeGrey5"
+      "description": "darkModeGrey4"
     },
     "textAppBarSelected": {
-      "value": "{palette.darkModeGrey2}",
+      "value": "{palette.movistarBlue}",
       "type": "color",
-      "description": "darkModeGrey2"
+      "description": "movistarBlue"
     },
     "customTabsBackground": {
       "value": "{palette.darkModeBlack}",
       "type": "color",
       "description": "darkModeBlack"
     },
     "tagTextPromo": {
-      "value": "{palette.purple40}",
+      "value": "{palette.purple35}",
       "type": "color",
-      "description": "purple40"
+      "description": "purple35"
     },
     "tagTextActive": {
       "value": "{palette.movistarBlue}",
       "type": "color",
       "description": "movistarBlue"
     },
     "tagTextInactive": {
-      "value": "{palette.darkModeGrey5}",
+      "value": "{palette.darkModeGrey3}",
```

Comment:
@hector are we sure with this change?
![image](https://github.com/Telefonica/mistica-design/assets/6722153/06236d26-1fcf-48bf-b674-7991b07257f1)